config/confighttp/confighttp.go

```diff
@@ -103,8 +104,7 @@ func (interceptor *headerRoundTripper) RoundTrip(req *http.Request) (*http.Respo
 }
 
 type HTTPServerSettings struct {
-	// Endpoint configures the listening address for the server.
-	Endpoint string `mapstructure:"endpoint"`
```

Comment:
This is probably breaking quite a few components in the contrib repo, right?

Reply:
Yes, need to synchronize changes